internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -381,6 +376,14 @@ func (t *Translator) translateBackendTrafficPolicyForRoute(policy *egv1a1.Backen
 			for _, r := range http.Routes {
 				// Apply if there is a match
 				if strings.HasPrefix(r.Name, prefix) {
+					if errs != nil {
```

Comment:
hey @MdSahil-oss can you raise another PR to address this for tcp and udp routes as well ?